raiden/tests/smart_contracts/netting_channel/test_updatetransfer.py

```diff
@@ -134,41 +170,56 @@ def test_update_must_fail_with_a_nonparticipant_transfer(tester_channels, privat
     nonparticipant_sign_key = PrivateKey(nonparticipant_key)
 
     transfer_nonparticipant.sign(nonparticipant_sign_key, nonparticipant_address)
-    transfer_nonparticipant_data = str(transfer_nonparticipant.packed().data)
 
-    nettingchannel.close('', sender=pkey0)
+    nettingchannel.close(sender=pkey0)
 
+    transfer_nonparticipant_hash = sha3(transfer_nonparticipant.packed().data[:-65])
```

Comment:
Since we use this hash now in many places for the message, perhaps make a method on the transfers that returns the hash, so that you don't have to explicitly always ask for the slice of the first 65 bytes of data?